### PR TITLE
Add proto definitions for GPU fallbacks

### PR DESCRIPTION
**Pull request by irfansharif**

### CODE REVIEW

**Comment by irfansharif** [comment]
@prbot approve

File: modal_proto/api.proto [hunk 1]
```diff
@@ -2072,6 +2073,11 @@ message TaskStats {
   double started_at = 4;
 }
 
+message TaskTemplate {
+  uint32 priority = 1;
+  Resources resources = 2;
+  uint32 concurrent_inputs = 3;
```
Comment (mwaskom) [inline modal_proto/api.proto:2079]: how is this going to eventually align with the new target/max concurrency parameterization?

**Review by mwaskom — COMMENTED** [review]
lgtm

File: modal_proto/api.proto (hunk 1 above)
Comment (irfansharif) [reply modal_proto/api.proto:2079]: https://github.com/modal-labs/modal-client/pull/2184 (`concurrent_inputs` is supposed to be the `target_` variant).

**Pull request merged**